### PR TITLE
Feature/12 bob and spin

### PR DESCRIPTION
**Pull request by packerbacker8**
This feature allows a script to be placed on a game object and allow it to bob and spin to desired parameters specified in the script.

### CODE REVIEW

File: Assets/Scripts/Utilities/Tags.cs [hunk 1]
```diff
@@ -7,4 +7,6 @@ public static class Tags
     public const string PLAYER_TAG = "Player";
 
     public const string COLLECTABLE_TAG = "Collectable";
+
+    public const string ENVIRONMENT_TAG = "Environment";
```
Comment (overseer25) [inline Assets/Scripts/Utilities/Tags.cs]: What is this?

File: Assets/Scripts/ConsoleCommands/TeleportCommand.cs [hunk 2]
```diff
@@ -95,9 +95,7 @@ public override void RunCommand(List<string> args = null)
                     }
                     catch (UnityException e)
                     {
-                        DeveloperConsole.Instance.AddMessageToConsole(e.Message, LogLevel.Error);
```
Comment (overseer25) [inline Assets/Scripts/ConsoleCommands/TeleportCommand.cs]: I appreciate refactoring the code to make it cleaner, but let's be careful making changes unrelated to the card in the future

File: Assets/Scripts/Utilities/Tags.cs (hunk 1 above)
Comment (packerbacker8) [reply Assets/Scripts/Utilities/Tags.cs]: Some items in my level were marked as Environment for moving objects around. Kept it around but we may rename things as we need.

File: Assets/Scripts/ConsoleCommands/TeleportCommand.cs (hunk 2 above)
Comment (packerbacker8) [reply Assets/Scripts/ConsoleCommands/TeleportCommand.cs]: Agreed. Just something I thought Id do as I saw it when the project opened. But in the future keep stuff card related is important.

**Pull request merged**